src/styles/styles.css

```diff
@@ -0,0 +1,316 @@
+@charset "UTF-8";
+
+/* VARIABLES HEADER */
+
+/* VARIABLES CARD */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+/* #region HEADER */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.header__nav {
+  padding: 0;
+}
+
+.header__nav-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.header__nav-item {
+  margin-right: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header__nav-item:last-child {
+  margin-right: 0;
+}
+
+.header__nav-item__link {
+  color: #060b35;
+  text-decoration: none;
+  display: block;
+  line-height: 60px;
+}
+
+.header__nav-item__link:hover,
+.header__nav-item__link:active {
+  color: #00acdc;
+}
+
+.header__nav-item__link:active {
+  text-decoration: underline;
+}
+
+.header__nav-item__link.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.header__nav-item__link.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 2px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.header__logo-link,
+.header__logo-link img {
+  width: 40px;
+  height: auto;
+  display: block;
+}
+
+/* #endregion HEADER */
+
+/* #region CATALOG */
+
+.catalog {
+  display: grid;
+  width: 944px;
+  grid-template-columns: repeat(4, 200px); /* 4 колонки по 200px */
+  justify-content: center;
+  gap: 46px 48px;
+  margin: 50px auto;
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    width: 1024px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .catalog {
+    width: 1200px;
+  }
+}
+
+/* #endregion CATALOG */
+
+/* #region CARD */
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card:hover {
+  transform: scale(1.2);
+  transition: transform 300ms ease;
+}
+
+.card__content {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+  height: 134px;
+  background-image: url(../images/imac.jpeg);
+  margin-left: 19px;
+  margin-top: 32px;
+  background-size: cover;
+}
```

Comment:
The `background-image: url(../images/imac.jpeg);` property could be replaced with an `img` tag in the HTML. This would improve accessibility because you can add an `alt` attribute to the `img` tag.